packages/inventory/src/components/table/EntityTableToolbar.js

```diff
@@ -296,9 +296,11 @@ const EntityTableToolbar = ({
     return <Fragment>
         <PrimaryToolbar
             {...props}
-            bulkSelect={{
-                ...bulkSelect,
-                isDisabled: bulkSelect?.isDisabled || !hasAccess
+            {...bulkSelect && {
```

Comment:
why is there a spread operator? :thinking:

Reply:
So we don't populate the `bulkSelect` at all if no value is passed. Without it this component could have send `bulkSelect={}` or (which is the point of this PR) `bulkSelect={{ isDisabled: false }}`.

Reply:
You can actually see in the failing tests that previsouly it was passing `bulkSelect` with disabled = false and that was causing this exact error.

Reply:
Oh.. I am wrong the whole time, I thought it was `(...bulkSelect) && {}` :facepalm: :facepalm: 

I am used to write these conditions as `...(bulkSelect && {})`

Sorry!!! :service_dog: